Web_Development/FrontEnd/Html_Accessibility.md

```diff
@@ -0,0 +1,84 @@
+# HTML Accessibility
+This provides the user to navigate and interact in a better way with your website . That's why make your HTML code as semantic as possible.
```

Comment:
@Pranshu321 Try to frame this in a better way so that it is more understandable.

Reply:
Ok